src/main/java/seedu/address/ui/PersonInfoPanel.java

```diff
@@ -103,8 +109,27 @@ private void loadPerson(ReadOnlyPerson person) {
         });
 
         loadPhoto(person);
+
+        currentlyViewedPerson = person;
+        logger.info("Currently Viewing: " + currentlyViewedPerson.getName() );
+    }
```

Comment:
And a person is assigned to be the currentlyViewedPerson when it is loaded in the app.